Source/ExtendedSampler.h

```diff
@@ -43,7 +43,7 @@ class ExtendedSound : public SynthesiserSound
     /** Returns the audio sample data.
         This could return nullptr if there was a problem loading the data.
     */
-    AudioBuffer<float>* getAudioData() const noexcept       { return data.get(); }
+    const AudioBuffer<float>& getAudioData() const noexcept       { return *data.get(); }

```

Comment:
This class was designed by JUCE team, I think data cannot be null because it comes from reader in constructor of Sound object, if reader cannot read data I think there will be some error or assert.